run_checks.py

```diff
@@ -12,35 +13,45 @@
 
 # create an instance of the API class
 api_instance = fingerprint_pro_server_api_sdk.FingerprintApi(configuration)
-visitor_id = os.environ["VISITOR_ID"]
-request_id = os.environ["REQUEST_ID"]
 
+end = int(datetime.now().timestamp() * 1000)
+start = int((datetime.now() - timedelta(days=90)).timestamp() * 1000)
+
+
+# FingerprintApi->search_events usage example
 try:
-    visits_response = api_instance.get_visits(visitor_id, limit=2)
-    pagination_key = visits_response.pagination_key
-    print("\n\n\nVisits response: \n", visits_response)
+    search_events_response = api_instance.search_events(2, start=start, end=end)
+    first_event = search_events_response.events[0]
+    first_event_identification_data = first_event.products.identification.data
+    visitor_id = first_event_identification_data.visitor_id
+    request_id = first_event_identification_data.request_id
+    print("\n\n\nSearch events response: \n", search_events_response)
+    search_events_response_second_page = api_instance.search_events(2, start=start, end=end, pagination_key=search_events_response.pagination_key)
 
-    visits_response = api_instance.get_visits(
-        visitor_id, limit=2, pagination_key=pagination_key)
+    if len(search_events_response_second_page.events) == 0:
+        print("Second page of FingerprintApi->search_events: is empty")
+        exit(1)
 
 except ApiException as e:
-    print("Exception when calling DefaultApi->visitors_visitor_id_get: %s\n" % e)
+    print("Exception when calling FingerprintApi->search_events: %s\n" % e)
     exit(1)
 
+# Use existing visitor_id from FingerprintApi->search_events response to check FingerprintApi->get_visits method
 try:
-    events_response = api_instance.get_event(request_id)
-    print("\n\n\nEvent response: \n", events_response.products)
+    visits_response = api_instance.get_visits(visitor_id, limit=2)
+    print("\n\n\nVisits response: \n", visits_response)
 
 except ApiException as e:
-    print("Exception when calling DefaultApi->get_event: %s\n" % e)
+    print("Exception when calling FingerprintApi->get_visits: %s\n" % e)
```

Comment:
nit: `->` is usually associated with PHP (at least for me 😁). We could do `FingerprintApi.get_visits()` here and in other places